packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -331,6 +334,27 @@ abstract class AnimatedWidgetBaseState<T extends ImplicitlyAnimatedWidget> exten
   void forEachTween(TweenVisitor<dynamic> visitor);
 }
 
+/// A base class for widgets with implicit animations that need to rebuild their
+/// Widget tree as the animation runs.
```

Comment:
lowercase widget.

We refer to the "widget tree" pretty loosely. In practice what that really means is the Element tree. We avoid referring to it as the Widget tree since it's not technically a tree of Widget objects.